docs/DeveloperGuide.md

```diff
@@ -753,6 +778,50 @@ The `ArgumentMultimap` utilises a `HashMap` to store an `ArrayList<String>` of a
 
 The `Prefix` is an `enum` consisting of `n/` ,`a/`, `p/`, `t/`, `e/`, `g/`, `m/` and a blank `Prefix` which is an empty String. The Prefixes listed previously correspond to [Name](#name), [Address](#address), [Phone](#phone), [Telegram Handle](#telegram-handle), [Email](#email), [Group Tags](#group-tag) and [Module Tags](#module-tag)).
 
+## **Recommenders**
+
+**API** : `Recommender.java` {to be filled in}
+
+<img src="images/RecommenderClass.png" style="width:80%;margin:0 10%">
+<div style="width:80%;margin:0 10%;text-align:center">
+    <b>Figure 4.6</b> Class Diagram for Recommender Module
+</div>
+<br>
+
+The `Recommender` component,
+* consists of 2 sub-components (modules) : `LocationRecommender` and `TimingRecommender`
+* recommends timing and location of meetups for relevant participants and the user.
+
+How the `Recommender` Component works:
+1. User enters a `meet/eat/study` command.
+2. Triggers the `TimingRecommender` to recommend common available timings amongst users and participants.
+3. `TimingRecommender` passes the recommended timings to `LocationRecommender`
+4. `LocationRecommender` recommends optimal meeting points paired with suitable timings based on
+   their `Location` at that particular timing.
```

Comment:
Hmm any reason why this is on a separate line? Cos in the actual DG I dont think it will turn out as intended. But can always change these minor things in 1.4.

Reply:
see my page in the PR description. It works

Reply:
If you are referring to line 800, yea, I know it doesn't work. I just want to wrap around.

Reply:
Alright, these things are small and can be handled later. It’s just that the CS2101 is peer review.